openwisp_users/base/models.py

```diff
@@ -178,14 +177,6 @@ def has_permission(self, permission):
         return permission in self.permissions
 
 
-# on django 3.1, the max length of first_name has been changed, we need to
-# backport this change to the previous versions to avoid migration issues
-# TODO: remove this when support to django < 3.1 is dropped
-if django.VERSION < (3, 1):
-    first_name = AbstractUser._meta.get_field('first_name')
-    first_name.max_length = 150
```

Comment:
just figured out this removal looks incorrect, because we need to keep supporting django 3.0, is this shim really needed also for django 3.0? @devkapilbansal could you check please?

Reply:
Yes, @nemesisdesign .If we need to keep supporting django 3.0, then we have to keep it. I am reverting it for now